File: aemet/SergioDeIscar-aemet-java/SergioDeIscar-aemet-java/src/main/java/org/example/dto/InformesDto.java
package org.example.dto;

import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.util.List;

@Root(name = "informes")
public class InformesDto {
    @ElementList(name = "informe", inline = true)
    private final List<InformeDto> informes;

    public InformesDto(List<InformeDto> informesDto){
        this.informes = informesDto;
    }

    //Getters
    public List<InformeDto> getInformes(){
        return informes;
    }
}

File: aemet/SergioDeIscar-aemet-kotlin/SergioDeIscar-aemet-kotlin/src/main/kotlin/dto/InformesDto.kt
package dto

import org.simpleframework.xml.ElementList
import org.simpleframework.xml.Root

@Root(name = "informes")
data class InformesDto(
    @field:ElementList(name = "informe", inline = true)
    @param:ElementList(name = "informe", inline = true)
    val informes: List<InformeDto>
)
